test/unit/organisations_fetcher_test.rb

```diff
@@ -20,10 +20,14 @@ class OrganisationsFetcherTest < ActiveSupport::TestCase
       :organisation,
       name: 'Ministry Of Misery',
       slug: slug,
+      content_id: "a1b2c3",
```

Comment:
I think it would be good to use a well-formed content-id here just to avoid any confusion.